<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWindow</class>
 <widget class="QDialog" name="PreviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreviewWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="previewArea">
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="monthToPreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Febuary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>March</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Apirl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>December</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monthPrevios">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monthNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
